src/iris/bin/sender.py

```diff
@@ -1462,6 +1474,12 @@ def message_send_enqueue(message):
 
     message_mode = message.get('mode')
     if message_mode and message_mode in per_mode_send_queues:
+        # check for known corner case limitations and correct accordingly
+        if message['mode'] == 'call':
```

Comment:
When we make changes like this, we have an `auditlog.message_change()` function that we call to indicate that a change has occurred.

Reply:
added